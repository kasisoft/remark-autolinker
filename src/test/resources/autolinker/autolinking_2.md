---
title: Some links, case sensitive
---
Some links:

* https://www.bibo.com
* Titled Link
* mailto--
* titled linK
