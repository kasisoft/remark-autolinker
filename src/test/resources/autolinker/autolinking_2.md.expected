---
title: Some links, case sensitive
---

Some links:

* https://www.bibo.com
* [Titled Link](https://www.gollum.test)
* [mailto--](mailto://frog@insect.com)
* titled linK
